docs/dictionary/command/visual-effect.lcdoc

```diff
@@ -15,21 +15,21 @@ OS: mac, windows, linux, ios, android, html5
 Platforms: desktop, mobile
 
 Example:
-visual effect dissolve
+visual effect "dissolve"
 go next card
 
 Example:
 -- This handler in an object will cause to hide then show 
 --  with a visual effect when clicked.
 on mouseUp
   hide me
-  show me with visual effect iris close slow
+  show me with visual effect "iris" close slow
```

Comment:
Hi Sam and Sean,

Thanks for the PR and the comments. Here are some clarifications, for anyone following this PR:

1. The `hide` and `show` commands can be used with a visual effect
2. There are 2 kinds of visual effects:
- The built-in ones, as described in the `visual effect` entry
- The QuickTime-specific ones, which were available only in the platforms were QT was supported (Win and OS X), and those effects were the ones returned by the `qtEffects()` function.

3. The `hide` command mentions that QT is deprecated in LC 8.1, thus the QT-specific effects should not be used with this command after LC 8.1

4. The `hide` command is currently marked as "deprecated". This is incorrect.
5. The `show` command does not mention anything about QT-specific effects being deprecated.

6. Some examples in `hide` and `show` entries have no quotes in the visual effect, which will cause a compile error when in Strict Compilation Mode.


So, we will file a new report about those issues in `hide` and `show` entries. So in short, we have to:

- Remove the deprecation note from `hide`
- Add a note in `show` about QT (as it is in `hide`
- Add quotes in the effects used in the example of `hide` and `show`
- Add references to `hide` and `show` in this PR (—> This is now done)